src/app/(ig)/IgDetail.module.css

```
@@ -541,27 +289,113 @@ body {
+}
+
+.compillingText .dots::after {
+  content: '';
+  display: inline-block;
+  width: 1.5ch;
+  overflow: hidden;
+  vertical-align: bottom;
+  animation: CompillingDots 1.4s steps(4, end) infinite;
+}
+
+@keyframes IgSpin {
```

Stylelint (17.14.0)

[error] Expected keyframe name "IgSpin" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)